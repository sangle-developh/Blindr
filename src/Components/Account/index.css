.account-page {
  height: 100%;
  width: 100%;
  display: flex;
  align-items: center;
  align-content: center;
  justify-content: center;
}
.wrapper-box {
  background-color: #e5e5e5;
  height: 80%;
  width: 80%;
  border-radius: 20px;
  margin: 64px;
  display: flex;
}

.box-column {
  flex: 1 1;
  background-color: bisque;
  border: 1px black solid;
}

.form-field {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  margin: 3rem 100px 2rem 100px;
}

.form-field > h3 {
  font-size: 22px;
}

.formControl {
  width: 195px;
  height: 32px;
}

.sex-orientation-input {
  width: 100%;
}

.sex-orientation-field {
  flex-direction: column;
  align-items: flex-start;
}

h3 {
  font-size: 22px;
}

.pronouns-input {
  width: 195px;
  height: 32px;
  background-color: transparent;
  border: 1px solid;
  color: black;
  font-size: 18px;
  padding: 0.2rem 0.7rem 0.2rem 0.7rem;
  border-radius: 15px;
}

.pronouns-input .MuiOutlinedInput-root {
  border-radius: 15px;
}

.selection {
  background-color: transparent;
  border: 1px solid;
  border-radius: 15px;
  color: black;
  font-size: 18px;
  padding: 0.2rem 0.7rem 0.2rem 0.7rem;
}

.connect-btn-container {
  display: flex;
  flex-direction: column;
  align-items: center;
  margin: 3rem;
}

.connect-btn {
  width: 374px;
  height: 51px;
  background-color: transparent;
  border: 2px solid;
  font-size: 22px;
  margin: 5px;
}
