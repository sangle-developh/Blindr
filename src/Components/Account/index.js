import React from 'react';
import './index.css';

const Account = () => {
  // const [value, setValue] = useState(50);
  return (
    <div className="account-page">
      <div className="wrapper-box">
        <div className="box-column">left</div>
        <div className="box-column">right</div>
      </div>
    </div>
  );
};

export default Account;
