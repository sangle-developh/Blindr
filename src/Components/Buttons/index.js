import React from 'react';

const Button = () => <button data-testid="button">Hello</button>;

export default Button;
