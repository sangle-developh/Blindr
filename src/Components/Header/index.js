import React, { useState } from 'react';

function Header() {
  const [menuActive, setMenuState] = useState(false);

  const toggleMobileMenu = () => {
    setMenuState(!menuActive);
  };

  return (
    <div>
      <nav className="navbar is-black" role="navigation" aria-label="main navigation">
        <div className="navbar-brand">
          <a className="navbar-item" href="/">
            <img src="https://bulma.io/images/bulma-logo.png" width="112" height="28" alt="Logo" />
          </a>

          <a
            role="button"
            className="navbar-burger"
            aria-label="menu"
            aria-expanded="false"
            data-target="navbarBasicExample"
            onKeyDown={toggleMobileMenu}
            onClick={toggleMobileMenu}
            tabIndex={0}
          >
            <span aria-hidden="true" />
            <span aria-hidden="true" />
            <span aria-hidden="true" />
          </a>
        </div>

        <div className={`navbar-menu ${menuActive ? 'is-active' : ''}`}>
          <div className="navbar-start">
            <a className="navbar-item">Home</a>

            <a className="navbar-item">FAQs</a>
          </div>

          <div className="navbar-end">
            <div className="navbar-item">
              <div className="buttons">
                <a className="button is-link">
                  <strong>Sign in</strong>
                </a>
              </div>
            </div>
          </div>
        </div>
      </nav>
    </div>
  );
}

export default Header;
