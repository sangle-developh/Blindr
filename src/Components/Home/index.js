import React from 'react';

export default () => {
  return (
    <div className="form">
      <p> HOMEPAGE</p>
    </div>
  );
};
