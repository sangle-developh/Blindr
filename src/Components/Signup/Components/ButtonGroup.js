import React from 'react';

import { Grid, Button } from '@material-ui/core';

const ButtonGroup = () => {
  return (
    <>
      <Grid item xs={12}>
        <Button
          variant="contained"
          type="submit"
          fullWidth
          style={{ color: '#E6A698', backgroundColor: '#332F2F' }}
        >
          CREATE ACCOUNT
        </Button>
      </Grid>
      <Grid item xs={12}>
        <Button
          variant="outlined"
          fullWidth
          style={{ color: '#332F2F', border: '1px solid #332F2F' }}
        >
          CANCEL
        </Button>
      </Grid>
    </>
  );
};

export default ButtonGroup;
