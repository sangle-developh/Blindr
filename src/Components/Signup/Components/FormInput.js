import React, { useState } from 'react';

import {
  Grid,
  Slider,
  InputLabel,
  TextField,
  Select,
  MenuItem,
  FormControl,
} from '@material-ui/core';

import { makeStyles, withStyles } from '@material-ui/styles';

const useStyles = makeStyles({
  logo: {
    width: '35%',
  },
  input: {
    [`& fieldset`]: {
      borderRadius: '12px',
      border: '1px solid #332F2F',
    },
  },
  select: {
    [`& fieldset`]: {
      color: '#332F2F',
      borderRadius: '12px',
      border: '0.5px solid #332F2F',
    },
  },
  createAccountButton: {
    color: '#E6A698',
    backgroundColor: '#332F2F',
    '&:hover': {
      color: '#332F2F',
      backgroundColor: 'white',
      border: '1px solid #332F2F',
      boxShadow: 'none',
    },
  },
  cancelButton: {
    color: '#332F2F',
    border: '1px solid #332F2F',
    '&:hover': {
      backgroundColor: 'white',
      border: '1px solid #332F2F',
      boxShadow: 'none',
    },
  },
});

const CustomSlider = withStyles({
  rail: {
    color: '#332F2F',
  },
  track: {
    color: '#332F2F',
  },
  thumb: {
    color: '#a566d6',
    border: '0.5px solid #332F2F',
  },
})(Slider);

const defaultValues = {
  username: '',
  dateOfBirth: undefined,
  gender: '',
  interest: '',
  minAge: 0,
  maxAge: 10,
  maxDistance: 0,
};

const FormInput = () => {
  const classes = useStyles();
  const [formValues, setFormValues] = useState(defaultValues);
  const handleInputChange = (e) => {
    const { name, value } = e.target;
    setFormValues({
      ...formValues,
      [name]: value,
    });
  };
  const handleSelectChange = (e) => {
    const { name, value } = e.target;
    setFormValues({
      ...formValues,
      [name]: value,
    });
  };
  const handleSliderChange = (name) => (e, value) => {
    setFormValues({
      ...formValues,
      [name]: value,
    });
  };
  return (
    <>
      <Grid item xs={12} sm={4}>
        <InputLabel htmlFor="label">Username</InputLabel>
      </Grid>
      <Grid item xs={12} sm={8}>
        <TextField
          id="username-input"
          name="username"
          variant="outlined"
          type="text"
          size="small"
          fullWidth
          autoComplete="off"
          value={formValues.username}
          onChange={handleInputChange}
          className={classes.input}
        />
      </Grid>
      <Grid item xs={12} sm={4}>
        <InputLabel htmlFor="label">D.O.B</InputLabel>
      </Grid>
      <Grid item xs={12} sm={8}>
        <TextField
          id="dob-input"
          name="dob"
          variant="outlined"
          type="date"
          size="small"
          fullWidth
          value={formValues.dateOfBirth}
          onChange={handleInputChange}
          className={classes.input}
        />
      </Grid>
      <Grid item xs={12} sm={4}>
        <InputLabel htmlFor="label">Gender</InputLabel>
      </Grid>
      <Grid item xs={12} sm={8}>
        <FormControl fullWidth>
          <Select
            id="gender-select"
            name="gender"
            size="small"
            fullWidth
            value={formValues.gender}
            onChange={handleSelectChange}
            className={classes.select}
          >
            <MenuItem key="man" value="man">
              Man
            </MenuItem>
            <MenuItem key="woman" value="woman">
              Woman
            </MenuItem>
            <MenuItem key="other" value="other">
              Other
            </MenuItem>
          </Select>
        </FormControl>
      </Grid>
      <Grid item xs={12} sm={4}>
        <InputLabel htmlFor="label">Interest</InputLabel>
      </Grid>
      <Grid item xs={12} sm={8}>
        <FormControl fullWidth>
          <Select
            id="interest-select"
            name="interest"
            size="small"
            fullWidth
            value={formValues.interest}
            onChange={handleSelectChange}
            className={classes.select}
          >
            <MenuItem key="men" value="men">
              Men
            </MenuItem>
            <MenuItem key="women" value="women">
              Women
            </MenuItem>
            <MenuItem key="everyone" value="everyone">
              Everyone
            </MenuItem>
          </Select>
        </FormControl>
      </Grid>

      <Grid item xs={12}>
        <InputLabel>Max Distance</InputLabel>
      </Grid>
      <Grid item xs={12}>
        <CustomSlider
          value={[0, formValues.maxDistance]}
          onChange={handleSliderChange('maxDistance')}
          valueLabelDisplay="auto"
          aria-labelledby="range-slider"
          className={classes.slider}
        />
      </Grid>
      <Grid item xs={12}>
        <InputLabel>Age Range</InputLabel>
      </Grid>
      <Grid item xs={12}>
        <CustomSlider
          value={[formValues.minAge, formValues.maxAge]}
          onChange={handleSliderChange}
          valueLabelDisplay="auto"
          aria-labelledby="range-slider"
          className={classes.slider}
        />
      </Grid>
    </>
  );
};

export default FormInput;
