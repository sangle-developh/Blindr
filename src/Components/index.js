export { default as Signup } from './Signup';
export { default as Header } from './Header';
export { default as Home } from './Home';
export { default as Account } from './Account';
