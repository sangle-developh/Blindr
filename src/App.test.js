import React from 'react';
import renderer from 'react-test-renderer';
import { Signup } from './Components';

test('Renders Sign up', () => {
  const component = renderer.create(<Signup />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
