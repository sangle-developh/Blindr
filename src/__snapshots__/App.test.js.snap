// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Sign up 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-grid-xs-4"
  />
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-7 MuiGrid-item MuiGrid-direction-xs-column MuiGrid-grid-xs-4"
  >
    <img
      alt="Logo"
      src="bindy-logo-white.png"
    />
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-align-items-xs-center"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated"
          htmlFor="label"
        >
          Username
        </label>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
      >
        <div
          className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
        >
          <div
            className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
              disabled={false}
              id="input"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              required={false}
              type="text"
            />
            <fieldset
              aria-hidden={true}
              className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
              style={
                Object {
                  "paddingLeft": 8,
                }
              }
            >
              <legend
                className="PrivateNotchedOutline-legend-2"
                style={
                  Object {
                    "width": 0.01,
                  }
                }
              >
                <span
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "&#8203;",
                    }
                  }
                />
              </legend>
            </fieldset>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-align-items-xs-center"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated"
          htmlFor="label"
        >
          Date of Birth
        </label>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
      >
        <div
          className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
        >
          <div
            className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
              disabled={false}
              id="input"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              required={false}
              type="text"
            />
            <fieldset
              aria-hidden={true}
              className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
              style={
                Object {
                  "paddingLeft": 8,
                }
              }
            >
              <legend
                className="PrivateNotchedOutline-legend-2"
                style={
                  Object {
                    "width": 0.01,
                  }
                }
              >
                <span
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "&#8203;",
                    }
                  }
                />
              </legend>
            </fieldset>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-align-items-xs-center"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated"
          htmlFor="label"
        >
          Gender
        </label>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
      >
        <div
          className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
        >
          <div
            className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
              disabled={false}
              id="input"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              required={false}
              type="text"
            />
            <fieldset
              aria-hidden={true}
              className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
              style={
                Object {
                  "paddingLeft": 8,
                }
              }
            >
              <legend
                className="PrivateNotchedOutline-legend-2"
                style={
                  Object {
                    "width": 0.01,
                  }
                }
              >
                <span
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "&#8203;",
                    }
                  }
                />
              </legend>
            </fieldset>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-align-items-xs-center"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated"
          htmlFor="label"
        >
          Interest
        </label>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
      >
        <div
          className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
        >
          <div
            className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              autoFocus={false}
              className="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
              disabled={false}
              id="input"
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              required={false}
              type="text"
            />
            <fieldset
              aria-hidden={true}
              className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
              style={
                Object {
                  "paddingLeft": 8,
                }
              }
            >
              <legend
                className="PrivateNotchedOutline-legend-2"
                style={
                  Object {
                    "width": 0.01,
                  }
                }
              >
                <span
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "&#8203;",
                    }
                  }
                />
              </legend>
            </fieldset>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-align-items-xs-center"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated"
        >
          Age Range
        </label>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
      >
        <span
          className="MuiSlider-root MuiSlider-colorPrimary"
          onMouseDown={[Function]}
        >
          <span
            className="MuiSlider-rail"
          />
          <span
            className="MuiSlider-track"
            style={
              Object {
                "left": "20%",
                "width": "17%",
              }
            }
          />
          <input
            type="hidden"
            value="20,37"
          />
          <span
            aria-labelledby="range-slider"
            aria-orientation="horizontal"
            aria-valuemax={100}
            aria-valuemin={0}
            aria-valuenow={20}
            aria-valuetext="20Â°C"
            className="MuiSlider-thumb MuiSlider-thumbColorPrimary PrivateValueLabel-thumb-5"
            data-index={0}
            onBlur={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onMouseLeave={[Function]}
            onMouseOver={[Function]}
            role="slider"
            style={
              Object {
                "left": "20%",
              }
            }
            tabIndex={0}
          >
            <span
              className="PrivateValueLabel-offset-7 MuiSlider-valueLabel"
            >
              <span
                className="PrivateValueLabel-circle-8"
              >
                <span
                  className="PrivateValueLabel-label-9"
                >
                  20
                </span>
              </span>
            </span>
          </span>
          <span
            aria-labelledby="range-slider"
            aria-orientation="horizontal"
            aria-valuemax={100}
            aria-valuemin={0}
            aria-valuenow={37}
            aria-valuetext="37Â°C"
            className="MuiSlider-thumb MuiSlider-thumbColorPrimary PrivateValueLabel-thumb-5"
            data-index={1}
            onBlur={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onMouseLeave={[Function]}
            onMouseOver={[Function]}
            role="slider"
            style={
              Object {
                "left": "37%",
              }
            }
            tabIndex={0}
          >
            <span
              className="PrivateValueLabel-offset-7 MuiSlider-valueLabel"
            >
              <span
                className="PrivateValueLabel-circle-8"
              >
                <span
                  className="PrivateValueLabel-label-9"
                >
                  37
                </span>
              </span>
            </span>
          </span>
        </span>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-align-items-xs-center"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated"
        >
          Max Distance
        </label>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
      >
        <span
          className="MuiSlider-root MuiSlider-colorPrimary"
          onMouseDown={[Function]}
        >
          <span
            className="MuiSlider-rail"
          />
          <span
            className="MuiSlider-track"
            style={
              Object {
                "left": "20%",
                "width": "17%",
              }
            }
          />
          <input
            type="hidden"
            value="20,37"
          />
          <span
            aria-labelledby="range-slider"
            aria-orientation="horizontal"
            aria-valuemax={100}
            aria-valuemin={0}
            aria-valuenow={20}
            aria-valuetext="20Â°C"
            className="MuiSlider-thumb MuiSlider-thumbColorPrimary PrivateValueLabel-thumb-5"
            data-index={0}
            onBlur={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onMouseLeave={[Function]}
            onMouseOver={[Function]}
            role="slider"
            style={
              Object {
                "left": "20%",
              }
            }
            tabIndex={0}
          >
            <span
              className="PrivateValueLabel-offset-7 MuiSlider-valueLabel"
            >
              <span
                className="PrivateValueLabel-circle-8"
              >
                <span
                  className="PrivateValueLabel-label-9"
                >
                  20
                </span>
              </span>
            </span>
          </span>
          <span
            aria-labelledby="range-slider"
            aria-orientation="horizontal"
            aria-valuemax={100}
            aria-valuemin={0}
            aria-valuenow={37}
            aria-valuetext="37Â°C"
            className="MuiSlider-thumb MuiSlider-thumbColorPrimary PrivateValueLabel-thumb-5"
            data-index={1}
            onBlur={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onMouseLeave={[Function]}
            onMouseOver={[Function]}
            role="slider"
            style={
              Object {
                "left": "37%",
              }
            }
            tabIndex={0}
          >
            <span
              className="PrivateValueLabel-offset-7 MuiSlider-valueLabel"
            >
              <span
                className="PrivateValueLabel-circle-8"
              >
                <span
                  className="PrivateValueLabel-label-9"
                >
                  37
                </span>
              </span>
            </span>
          </span>
        </span>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-align-items-xs-center"
    >
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-text"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          CREATE ACCOUNT
        </span>
      </button>
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-outlined"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          CANCEL
        </span>
      </button>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-grid-xs-4"
  />
</div>
`;
